stdlib/asyncio/events.pyi

```diff
@@ -113,13 +113,8 @@ class AbstractEventLoop:
     slow_callback_duration: float
     @abstractmethod
     def run_forever(self) -> None: ...
-    # Can't use a union, see mypy issue  # 1873.
```

Comment:
This comment was added in https://github.com/python/typeshed/pull/435, and https://github.com/python/mypy/issues/1873 has long since been resolved